opifancontrol.sh

```diff
@@ -116,13 +116,18 @@ while true; do
     fi
 
     if [ $TARGET_PWM -ne $CURRENT_PWM ]; then
+        BASE_RAMP_UP_DELAY_TS=$(date +%s)
         # If the fan is currently off, wait for the ramp up delay before turning it on to avoid rapid on/off cycles
-        if [ $TARGET_PWM -gt $CURRENT_PWM ] && [ $((LAST_RAMPED_DOWN_TS + $RAMP_UP_DELAY_SECONDS)) -gt $(date +%s) ]; then
-            TARGET_PWM=$CURRENT_PWM
+        if [ $TARGET_PWM -gt $CURRENT_PWM ] && [ $((LAST_RAMPED_DOWN_TS + $RAMP_UP_DELAY_SECONDS)) -gt $BASE_RAMP_UP_DELAY_TS ]; then
+            RAMP_UP_DELAY_REMAIN_SECONDS=$(($RAMP_UP_DELAY_SECONDS - $BASE_RAMP_UP_DELAY_TS + $LAST_RAMPED_DOWN_TS))
+            debug "Delay of $RAMP_UP_DELAY_REMAIN_SECONDS sec before turning on the fan ... Target PWM: $TARGET_PWM"
+            #TARGET_PWM=$CURRENT_PWM
```

Comment:
That should not be used, since it creates a problem addressed via https://github.com/jamsinclair/opifancontrol/pull/2, but what is worse, it does **not** introduce explicit delay for ramp-up (via sleep), but uses TEMP_POLL_SECONDS timer to accommodate required RAMP_DOWN_DELAY_SECONDS, that in turn introduces unnecessary PWM duty cycle changes - `gpio pwm $FAN_GPIO_PIN 0`:

```
Apr 06 14:13:39 orangepi5plus opifancontrol.sh[25384]: Changing Fan Speed | CPU temp: 46, target PWM: 0, current PWM: 0
Apr 06 14:14:43 orangepi5plus opifancontrol.sh[25384]: Changing Fan Speed | CPU temp: 49, target PWM: 0, current PWM: 0
Apr 06 14:15:45 orangepi5plus opifancontrol.sh[25384]: Changing Fan Speed | CPU temp: 51, target PWM: 0, current PWM: 0
Apr 06 14:16:47 orangepi5plus opifancontrol.sh[25384]: Changing Fan Speed | CPU temp: 53, target PWM: 0, current PWM: 0
Apr 06 14:17:49 orangepi5plus opifancontrol.sh[25384]: Changing Fan Speed | CPU temp: 50, target PWM: 0, current PWM: 0
Apr 06 14:18:51 orangepi5plus opifancontrol.sh[25384]: Changing Fan Speed | CPU temp: 54, target PWM: 0, current PWM: 0
Apr 06 14:19:53 orangepi5plus opifancontrol.sh[25384]: Changing Fan Speed | CPU temp: 49, target PWM: 0, current PWM: 0
```